Clarify default (#3870)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/sharepoint-azureb2b-integration.md

Once the integration is enabled you and your users don't have to reshare or do any manual migration for guests previously shared with. Instead, when someone outside your organization clicks on a link that was created before Azure AD B2B integration was enabled, SharePoint will automatically create a B2B guest account. This guest account is created for the user who originally created the sharing link. (If the user who created the link is no longer in the organization or no longer has permission to share, the guest won't be added to the directory and the file will need to be reshared.)

> SharePoint and OneDrive integration with the Azure AD B2B one-time passcode feature is currently not enabled by default, new tenants as of March 31, 2023 will have it enabled by default.

Reviewer: ```suggestion
SharePoint and OneDrive integration with the Azure AD B2B one-time passcode feature is currently not enabled by default, but it is enabled by default for new tenants as of March 31, 2023.
```
